source/solvers/heat_transfer_assemblers.cc

```diff
@@ -187,8 +263,15 @@ HeatTransferAssemblerCore<dim>::assemble_rhs(
              scratch_data.source[q] * phi_T_i) *
             JxW;
 
+          // Apply GLS
```

Comment:
See my other comment
```suggestion
          // Apply SUPG
```